src/components/atom/AppLinkButton.tsx

```diff
@@ -0,0 +1,17 @@
+import ButtonProps from "../../types/ButtonProps";
+import {FC} from "react";
+import {Link} from "react-router-dom";
+
```

Comment:
```suggestion
interface AppLinkButtonProps extends ButtonProps {
    toGo: string
}
```

I would create a new type only for this special kind of button where it is expected to always receive a `toGo` string